src/CellOutputViewer/transforms/Vega2.ts

```diff
@@ -0,0 +1 @@
+export { Vega2 as default } from "@nteract/transform-vega";
```

Comment:
>Vega2 [](http://example.com/codeflow?start=9&length=5)

Can you not export all the transforms from one single file?

do you really need a file per transform?

Reply:
In order to use `React.lazy` I had to declare separate exports with defaults. In this case it is `Vega2`. Another way is to use it like `React.lazy(() => import("@nteract/transform-vega").then(vega => { default: vega.Vega2 })`. Thought it is cleaner this way.